bower.json

```diff
@@ -19,6 +19,7 @@
     "requirejs-mustache": "https://github.com/jfparadis/requirejs-mustache.git#5fb8c0a3e8560526e8f9617a689e2924a8532d0a",
     "requirejs-text": "2.0.10",
     "sinon": "http://sinonjs.org/releases/sinon-1.7.1.js",
+    "sjcl": "1.0.2",
```

Comment:
Beyond the scope of this PR, but can sjcl in the fxa-js-client be exposed to eliminate the duplication?